shared/README.md

```diff
@@ -74,7 +74,25 @@ yarn run rn-gobuild-android
 react-native run-android
 ```
 
+### Debugging with React Developer Tools extension
 
+1) Install the [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) in your regular Chrome browser.
+2) Run a yarn command (e.g. `yarn run start`) with the
+`KEYBASE_LOCAL_DEBUG` and `KEYBASE_DEV_TOOL_EXTENSIONS` environment
+variables set appropriately. An example for macOS would be
+
+```
+env KEYBASE_SHOW_DEVTOOLS=1 KEYBASE_LOCAL_DEBUG=1 KEYBASE_DEV_TOOL_EXTENSIONS="$HOME/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/2.5.0_0" yarn run start
```

Comment:
Couple of things

1.  I wouldn't suggest inlining this before the command, i'd just suggest settings the environment variable
1. You don't need the actual version number in there, the script itself looks for the version , which also means you can remove the comment below about needing to update it when it changes